pkg/work/webhook/start.go

```diff
@@ -34,13 +32,9 @@ func (c *Options) RunWebhookServer() error {
 		Scheme:                 scheme,
 		HealthProbeBindAddress: ":8000",
 		WebhookServer: webhook.NewServer(webhook.Options{
-			TLSOpts: []func(config *tls.Config){
-				func(config *tls.Config) {
-					config.MinVersion = tls.VersionTLS13
-				},
-			},
-			Port:    c.Port,
-			CertDir: c.CertDir,
+			TLSMinVersion: "1.2",
```

Comment:
I think TLSMinVersion is deprecated, can we use TLSOpts instead?

Reply:
Updated the code to use TLSOpts.